src/main/java/com/daitangroup/api/controller/PersonController.java

```diff
@@ -1,35 +1,111 @@
 package com.daitangroup.api.controller;
 
 import com.daitangroup.api.model.Person;
-import com.daitangroup.api.repository.PersonRepository;
+import com.daitangroup.api.services.GarageService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.ResponseBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.Optional;
 
 @RestController
-@RequestMapping("/person")
+@RequestMapping("/persons")
 public class PersonController {
 
     @Autowired
-    private PersonRepository repository;
+    private GarageService garageService;
 
     @RequestMapping(method = RequestMethod.GET, value = "/{id}")
-    @ResponseBody
-    public ResponseEntity<Person> find(@PathVariable Long id) {
+    public ResponseEntity<Person> findPersonById(@PathVariable Long id) {
 
-        Optional<Person> person = repository.findById(id);
+        Optional<Person> personResult = garageService.findPersonById(id);
 
-        if (person.isPresent()) {
-            return new ResponseEntity<>(person.get(), HttpStatus.OK);
+        if (personResult.isPresent()) {
+            return new ResponseEntity<>(personResult.get(), HttpStatus.OK);
         }
 
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public ResponseEntity<Iterable> findAllPersons() {
+
+        Iterable<Person> personsResult = garageService.findAllPersons();
+
+        if (personsResult.iterator().hasNext()) {
+            return new ResponseEntity<>(personsResult, HttpStatus.OK);
+        }
+
+        return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "/cpf/{cpf}")
+    public ResponseEntity<Person> findPersonByCpf(@PathVariable String cpf) {
+
+        Optional<Person> personResult = garageService.findPersonByCpf(cpf);
+
+        if (personResult.isPresent()) {
+            return new ResponseEntity<>(personResult.get(), HttpStatus.OK);
+        }
+
+        return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/create")
+    public ResponseEntity<Person> createPerson(@RequestBody Person person) {
+
+        Person personSaved = garageService.createPerson(person);
+
+        if(personSaved != null){
+            return new ResponseEntity<>(person, HttpStatus.OK);
+        }
+
+        return new ResponseEntity<>(HttpStatus.NOT_ACCEPTABLE);
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/associatevehicle")
```

Comment:
It's ok to use CamelCase here. It's not wrong but it makes things easier to read

Do some experiments with the endpoint path and parameters' name

Reply:
Best practice is to use hiphen.

https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris